ui-modules/utils/logbook/logbook.js

```diff
@@ -91,8 +91,6 @@ export function logbook() {
         $scope.fieldsToShow = ['timestamp', 'class', 'message']
         $scope.logFields = [
             {name: 'Timestamp',   value: 'timestamp',  selected: true},
-            {name: 'Task ID',     value: 'taskId',     selected: false},
-            {name: 'Entity IDs',  value: 'entityIds',  selected: false},
```

Comment:
@jcabrerizo, there is no task ID or entity ID in logs.

Reply:
Have you tried to invoke an effector? Is how I teste it and the taks Id was there

Reply:
Ok, I see now, reverting back.
<img width="1267" alt="Screenshot 2021-07-20 at 19 19 59" src="https://user-images.githubusercontent.com/81319331/126375392-87950cd4-35ec-4f96-827c-72e5fd7a6069.png">